src/runtime/flags.ts

```diff
@@ -24,6 +24,7 @@ class FlagDefaults {
   static enforceRefinements = false;
   static useSlandles = false;
   static fieldRefinementsAllowed = false;
+  static defaultReferenceMode = false;
```

Comment:
Can you add a TODO which links to a GitHub issue for this?

Reply:
done